docs/developer/checkout.mdx

```diff
@@ -1071,8 +1072,53 @@ A response would look like this:
 }
 ```
 
+## Finalizing checkout with Saleor App
+:::caution
+
+This feature is currently in **Feature Preview**. This part of Saleor is not complete
```

Comment:
It is subject to change, but it is probably complete?

Reply:
This is a standard form that we're adding to all `Feature Preview`s sections. I can change it and remove `not complete` if you want

Reply:
thanks!